packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -244,6 +244,11 @@ async function runAxeTestsForBlockEditor() {
 			'.edit-post-layout__metaboxes',
 			// Ignores elements created by TinyMCE.
 			'.mce-container',
+			// These properties were not included in the 1.1 spec
+			// through error, they should be allowed on role="row":
+			// https://github.com/w3c/aria/issues/558
+			'[role="row"][aria-setsize]',
+			'[role="row"][aria-posinset]',
```

Comment:
This seems like it should allow the e2e tests to ignore the incorrect parts of the spec, but the tests still seem to be failing. 😕